presto-docs/src/main/sphinx/migration/from-hive.rst

```diff
@@ -109,3 +109,14 @@ Presto query::
     SELECT student, score
     FROM tests
     CROSS JOIN UNNEST(scores) AS t (score);
+
+Caution with datediff
+---------------------
+
+The Hive ``datediff`` function is declared as ``datediff(string enddate, string
+startdate)``. The equivalent function in ANSI SQL is ``date_diff`` and it is
```

Comment:
`date_diff` is a Presto specific function, not part of the SQL standard

Reply:
changed